torrent.go

```diff
@@ -1252,6 +1263,11 @@ func (t *Torrent) announceDHT(impliedPort bool) {
 					allAddrs[key] = struct{}{}
 				}
 				cl.mu.Lock()
+				if t.peers == nil { // torrent can be already closed. return.
```

Comment:
True. It additional runtime checks of proper behaviour of closed torrents. Should we call addPeer on closed torrent? I think no.